### ISSUE TITLE
View inheritance may not use attribute 'string' as a selector.

### ISSUE THREAD

**Issue by helgetan** [shingonoide/odoo_ezdoo#2]
Hey tried to install the maintenance mode module on odoo9 and got this error.
Any Help ?

```
Error details:
View inheritance may not use attribute 'string' as a selector.
Error context:
View Website Maintenance settings
[view_id: 438, xml_id: n/a, model: website.config.settings, parent_id: 289]
None" while parsing /mnt/extra-addons/website_maintenance/view/res_config.xml:4, near
<record id="view_website_maintenance_config_settings" model="ir.ui.view">
            <field name="name">Website Maintenance settings</field>
            <field name="model">website.config.settings</field>
            <field name="inherit_id" ref="website.view_website_config_settings"/>
            <field name="arch" type="xml">
                <group string="Menu" position="before">
                    <group string="Maintenance Mode">
                        <field name="maintenance_mode"/>
                        <field name="maintenance_message" placeholder="(Opcional) We're undergoing maintenance now, please come back later."/>
                    </group>
                </group>
            </field>
        </record>
```

**Comment by shingonoide** [comment]
Thanks for reporting and sorry I haven't try the module on v9 yet.
I'll see how to fix that.

**Comment by helgetan** [comment]
ok thanks for the help

**Comment by shingonoide** [comment]
This error was fixed by commit af40246c138f6b3137082d40e422137ecc2a1308
@helgetan thanks again for reporting.
